### DISCUSSION TITLE
"A database consistency issue has been caught" does anyone else get this error frequently?

### DISCUSSION THREAD

**Discussion by marklar666** [abraunegg/onedrive#2302] [General]
At least once a week or two I get this error:
```
Performing a database consistency and integrity check on locally stored data due to fullscan requirement ...
ERROR: A database consistency issue has been caught. A --resync is needed to rebuild the database.
onedrive.service: Main process exited, code=exited, status=255/EXCEPTION 
```
I have to stop the service, run onedrive with --resync attribute, it fixes the issue but a week or two later I get this error again. Is anyone experiencing this?

**Comment by abraunegg** [comment]
@marklar666 
What application version are you using? What is the output of `onedrive --version` ?

That specific error is indicative of you running 2 or more applications against the same database configuration.

**Comment by marklar666** [comment]
It's the latest v2.4.23-1+np1 but I've been experiencing this for a few months now.
There is no other instance of onedrive running on this machine. I start it with _systemctl --user start onedrive_ if that matters.

**Reply by abraunegg** [reply]
@marklar666 
> It's the latest v2.4.23-1+np1 but I've been experiencing this for a few months now. There is no other instance of onedrive running on this machine. I start it with _systemctl --user start onedrive_ if that matters.

You probably are hit with the Ubuntu packaging bug that existed for a long time - where a secondary systemd service was installed and created and it *is* running in the background - see: https://github.com/abraunegg/onedrive/issues/1274

The resolution is listed here: https://github.com/abraunegg/onedrive/issues/1274#issuecomment-813846904

**Reply by marklar666** [reply]
Thx. I will remove the symbolic link and will be starting onedrive manually from now on.

**Reply by abraunegg** [reply]
@marklar666 
If you still run into this issue post starting manually - you need to look at your underlying filesystem and how it handles writing data out to the local database when the 'onedrive' application exits.

If your home directory is using btrfs and/or encrypted - then you need to look at that as your starting point as something with your filesystem is not playing nice with this client.
